openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -215,6 +220,65 @@ def test_is_user_eligible_for_credit(self):
         is_eligible = api.is_user_eligible_for_credit('abc', credit_course.course_key)
         self.assertFalse(is_eligible)
 
+    def test_get_credit_requirement(self):
+        self.add_credit_course()
+        requirements = [
+            {
+                "namespace": "grade",
+                "name": "grade",
+                "display_name": "Grade",
+                "criteria": {
+                    "min_grade": 0.8
+                }
+            }
+        ]
+        requirement = get_credit_requirement(self.course_key, "grade", "grade")
+        self.assertIsNone(requirement)
+
+        set_credit_requirements(self.course_key, requirements)
+        requirement = get_credit_requirement(self.course_key, "grade", "grade")
+        self.assertIsNotNone(requirement)
+        self.assertEqual(requirement['namespace'], requirements[0]['namespace'])
```

Comment:
Why not use a stronger assertion comparing the value of `requirement` to a dictionary containing each expected value?